CHANGELOG.md

```diff
@@ -4,6 +4,127 @@
+
+**- feat(replay): Change the behavior of error-based sampling (#7768)**
+
+We have changed the behavior of error-based sampling, as well as adding & adjusting APIs a bit to be more aligned with expectations.
+
+First, the `replaysOnErrorSampleRate` will now be applied per-error at error-time, not in advance at session creation time -
+so the sampling rate was not accurate if you had sessions with many errors.
+Now whenever an error happens, we'll check the error sample rate, and potentially flush the session.
+
+Related to this, we also reworked the public `start()` method and added a new `startBuffering()` method.
+
+When calling `start()`, it will now _always_ start a session-based Replay, ignoring sample rates.
+Note that it will throw an error if a session is already running.
+
+You can use the new `startBuffering()` method to start a new Replay in buffering-mode.
+This will either wait for an error (when `replaysOnErrorSampleRate > 0`), or until you manually call `replay.flush()`.
+
+Note that with this change, as soon as either `replaysSessionSampleRate` or `replaysOnErrorSampleRate` are `> 0`,
+we will _always_ start recording in the background, either in session mode or in buffering mode.
+When both sample rates are set to `0`, you can manually start the session with either `replay.start()` or `replay.startBuffering()`.
+
+
+**- Improved Replay `flush()` & `stop()` APIs**
+  - feat(replay): Change `flush()` API to record current event buffer (#7743)
+  - feat(replay): Change `stop()` to flush and remove current session (#7741)
+
+We have revamped some existing public API of Replay in order to be better aligned with expectations.
+
+`replay.flush()` now works differently when in error sampling mode:
+It will send the buffered replay data, then convert the session to a "regular" session and continue recording.
+With this, you can manually start capturing at any point.
+
+`replay.stop()` will now flush the event buffer before stopping, and remove the session from session storage.
+This also means that `stop()` now returns a promise.
+
+- **feat(core): Add multiplexed transport (#7926)**
+
+We added a new transport to support multiplexing.
+With this, you can configure Sentry to send events to different DSNs, depending of a logic of your choosing:
+
+```js
+import { makeMultiplexedTransport } from '@sentry/core';
+import { init, captureException, makeFetchTransport } from '@sentry/browser';
+
+function dsnFromFeature({ getEvent }) {
+  const event = getEvent();
+  switch(event?.tags?.feature) {
+    case 'cart':
+      return ['__CART_DSN__'];
+    case 'gallery':
+      return ['__GALLERY_DSN__'];
+  }
+  return []
+}
+
+init({
+  dsn: '__FALLBACK_DSN__',
+  transport: makeMultiplexedTransport(makeFetchTransport, dsnFromFeature)
+});
+```
+
+- **fix(core): fix(utils): default normalize() to a max. of 100 levels deep instead of Inifnity (#7957)**
```

Comment:
m: I don't think this is necessary to highlight.

Reply:
@lforst pointed out specifically that we should highlight this - For me both ways are fine, also OK with leaving it under "other".